app/views/themes/reshare_show/hyrax/base/_metadata.html.erb

```diff
@@ -0,0 +1,5 @@
+<dl class="work-show <%= dom_class(presenter) %> mb-0" <%= presenter.microdata_type_to_html %>>
+  <%= render 'attribute_rows', presenter: presenter %>
+  <%= presenter.attribute_to_html(:embargo_release_date, render_as: :date, html_dl: true) %>
```

Comment:
I'm not sure embargo or leases will apply to reshare. we can probably remove these.